src/main/java/world/bentobox/bentobox/schems/Clipboard.java

```diff
@@ -413,15 +448,35 @@ private boolean copyBlock(Block block, Location copyOrigin, boolean copyAir, Col
             return true;
         }
 
+        // Block state
+        BlockState bs = block.getState();
+
         // Set block data
-        s.set("bd", block.getBlockData().getAsString());
+        if (bs.getData() instanceof Attachable) {
+            plugin.logDebug("Attachable");
+            ConfigurationSection a = blockConfig.createSection(ATTACHED + pos);
+            a.set("bd", block.getBlockData().getAsString());
+            // Placeholder
+            s.set("bd", "minecraft:stone");
```

Comment:
Attachment blocks need to be set to something that doesn't have to be attached, e.g. STONE. If you do not do this and there is water or lava there, then that liquid will flow between ticks and mess everything up. It could be changed to AIR I think maybe and still be the same.